style.css

```diff
@@ -0,0 +1,65 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
```

Comment:
Giving height into body is not professional try to wrap into Parent div or main tag instead of body